### ISSUE TITLE
Code fix for `NUnit2055` can generate invalid code for classic asserts

### ISSUE THREAD

**Issue by philipp-naused** [nunit/nunit.analyzers#953]
```cs
[Test]
public void SampleTest()
{
    object value = "Hello, World!";
    ClassicAssert.IsTrue(value is string);
}
```
This code will trigger two warnings:
`NUnit2003: Consider using the constraint model, Assert.That(expr, Is.True), instead of the classic model, ClassicAssert.IsTrue(expr)`
`NUnit2055: Consider using Is.InstanceOf<string> instead of 'is string' expression for better assertion messages`

I have this as `.editorconfig` to turn them both into warnings:
```ini
[*.cs]
dotnet_analyzer_diagnostic.category-Assertion.severity = warning
```

If I apply all code fixes (`dotnet format`), NUnit2055 will be applied first and generate this:
```cs
ClassicAssert.IsTrue(value, Is.InstanceOf<string>());
```
which doesn't compile...

If I explicitly tell it to fix NUnit2003 first, this doesn't happen.

**Comment by PhilippNaused** [comment]
I can reproduce it in both VS18 and vscode.

<img width="920" height="385" alt="Image" src="https://github.com/user-attachments/assets/0e266d8d-f5b5-4123-8f9e-2b5e68f7c14d" />

<img width="484" height="240" alt="Image" src="https://github.com/user-attachments/assets/199eaf05-86cd-419d-9733-29dfec690e52" />

`dotnet format` (sometimes?) fixes them in the right order.
But the "Fix all warning and errors set in EditorConfig" code cleanup in VS, fixes them in the wrong order.

Here is the minimal repo: (just 3 files)
[code.zip](https://github.com/user-attachments/files/23431756/code.zip)
